### PR TITLE
docs: Fix example of tx usage

### PR DESCRIPTION
**Pull request by acolytec3**
## Description

Fixes documentation of @ethereumjs/tx usage in the code example for `eth_sendSignedTransaction`
## Type of change

- [x] I have made corresponding changes to the documentation.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,35 +4,9 @@ All notable changes to this project will be documented in this file.
...
-### Fixed
-
-- I've fixed XY (#1000)
-
-### Security
-
-- I've improved the security in XY (#1000)
-
--->
+- Correct `eth_sendSignedTransaction` code example (#5402)
```
Comment (jdevcs) [inline CHANGELOG.md]: This should be placed under 1.7.6

**Review by jdevcs — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (acolytec3) [reply CHANGELOG.md]: To confirm, I should just add something like this?
```md
## [1.7.6]

### Changed
```

Comment (avkos) [reply CHANGELOG.md]: Not add. we already have this line
## [1.7.6] (line 588)
just place your changes under  this line

**Review by avkos — CHANGES_REQUESTED** [review]

**Comment by acolytec3** [comment]
@avkos Done!

**Review by avkos — APPROVED** [review]

File: docs/web3-eth.rst [hunk 2]
```diff
@@ -1664,9 +1664,9 @@ Example
     }
 
     var tx = new Tx(rawTx, {'chain':'ropsten'});
-    tx.sign(privateKey);
+    var signedTx = tx.sign(privateKey);
```
Comment (jdevcs) [inline docs/web3-eth.rst:1669]: current example code in our doc works fine, and sign doesn't return any thing, if you refer ethereumjs lib doc. so we dnt need this change/PR.  checked with "ethereumjs-tx": "^2.1.2"

Comment (acolytec3) [reply docs/web3-eth.rst:1669]: That repository was last updated 2 years ago and `ethereumjs-tx` is a deprecated module.  It has been replaced by [@ethereumjs/tx](https://www.npmjs.com/package/@ethereumjs/tx) which has the API represented by my PR.

Comment (acolytec3) [reply docs/web3-eth.rst:1669]: [This](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/tx/docs/classes/Transaction.md#sign) is the API for the currently supported method.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,36 +4,6 @@ All notable changes to this project will be documented in this file.
...
-### Fixed
-
-- I've fixed XY (#1000)
-
-### Security
-
-- I've improved the security in XY (#1000)
-
--->
-
```
Comment (jdevcs) [inline CHANGELOG.md]: This example template shouldn't be removed.

Comment (acolytec3) [reply CHANGELOG.md]: Sorry, removed it in error.  I've added it back

**Comment by jdevcs** [comment]
This PR is doc change so can merge without status of all CI actions.

**Pull request merged**